src/components/Footer.tsx

```diff
@@ -3,52 +3,50 @@ import ".././styles.css";
 
 // Footer with source code and project information that goes at the bottom of each page
 const Footer = (): ReactElement => {
-  return (
-    <div className="w-full px-2 py-4 mt-4 flex flex-row justify-between border-t-2 absolute bottom-0">
-      <div>
-        Made by students @{" "}
-        <a
-          className="link"
-          onClick={() => {
-            window.open("https://sandboxnu.com");
-          }}
-        >
-          Sandbox
-        </a>
-        . Source on{" "}
-        <a
-          className="link"
-          onClick={() => {
-            window.open("https://github.com/sandboxnu/sga-tooling");
-          }}
-        >
-          GitHub.
-        </a>
-      </div>
+	return (
```

Comment:
Should we decide on a default indentation and/or formatting for the entire project? Just so that we don't have huge file changes of just whitespace every time someone makes a PR.

Reply:
Good catch - I don't know why my VSCode decided to randomly start indenting things by tabs... I'll fix this and re-push.

Reply:
And to answer your question I think the formatting and indenting we have now works! I think all of us are using Prettier as well (hopefully) so I think the Prettier formatting can be standardized